site/content/docs/5.2/components/tags.md

```diff
@@ -0,0 +1,213 @@
+Most of the time, tags must be inside a list (`<ul>` or `<ol>`).
+
+{{< example >}}
+<ol class="list-unstyled d-flex gap-2 flex-wrap m-0">
+  <li><a class="tag" href="#">1. Introduction</a></li>
+  <li><a class="tag" href="#">2. Exposure</a></li>
+</ol>
+{{< /example >}}
+
+### Input
+
+This kind of tag are built on `<span>`.
+
+For a list of selected items use `<ul>` or `<ol>` (if tags are ordered). Here is an [accessible example](https://a11y-guidelines.orange.com/en/web/components-examples/tags/examples/filter-by-brand/) of a component that can be built on top of this variant.
+
+{{< example class="d-flex gap-2 align-items-center" >}}
+<ul class="list-unstyled d-flex gap-2 flex-wrap m-0">
+  <li><span class="tag" id="labelTag1">
+    Dismissible tag
+    <button class="close" aria-labelledby="labelTag1"><span class="visually-hidden">Close</span></button>
+  </span></li>
+  <li><span class="tag" id="labelTag2">
+    <svg width="1rem" height="1rem" fill="currentColor" aria-hidden="true" focusable="false">
+      <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+    </svg>
+    Dismissible tag
+    <button class="close" aria-labelledby="labelTag2"><span class="visually-hidden">Close</span></button>
+  </span></li>
+</ul>
+{{< /example >}}
+
+## Dark variant
+
+{{< callout info >}}
+We add an extra `<p>` around the `<span>` here for accessibility concerns.
+{{< /callout >}}
+
+Add `.tag-dark` to the `.tag` for a dark variant.
+
+{{< example class="d-flex gap-2 align-items-center bg-dark" >}}
+<p class="mb-0"><span class="tag tag-dark">Informative</span></p>
+<button class="tag tag-dark">Filter</button>
+<a class="tag tag-dark" href="#">Navigation</a>
+<p><span class="tag tag-dark" id="labelTag3">
+  <svg fill="currentColor" width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  Input
+  <button class="close" aria-labelledby="labelTag3"><span class="visually-hidden">Close</span></button>
+</span></p>
+{{< /example >}}
+
+## Sizes
+
+{{< callout info >}}
+We add an extra `<p>` around the `<span>` here for accessibility concerns.
+{{< /callout >}}
+
+Add `.tag-sm` to the `.tag` for a small variant.
+
+{{< example class="d-flex gap-2 align-items-center" >}}
+<h3 class="visually-hidden">Small tag variant</h3>
+<p class="mb-0"><span class="tag tag-sm">Informative</span></p>
+<button class="tag tag-sm">Filter</button>
+<a class="tag tag-sm" href="#">Navigation</a>
+<p><span class="tag tag-sm" id="labelTag4">
+  <svg fill="currentColor" width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"/>
+  </svg>
+  Input
+  <button class="close" aria-labelledby="labelTag4"><span class="visually-hidden">Close</span></button>
```

Comment:
Are we sure about this rendering? Focus outline seems very close to the text.
![Screenshot 2022-10-20 at 07 53 34](https://user-images.githubusercontent.com/17381666/196867231-4fdfe0d7-5091-4bf2-bf46-49bf2a25cb4d.png)

Reply:
Seen with design, we keep it as-is!